<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="GURLPG02P17R" failures="3" tests="9" name="Personal Loan" time="0.14" errors="0" timestamp="2024-06-22T17:14:28 IST">
  <testcase classname="testNG.TestNGAnnotations" name="login" time="0.008"/>
  <testcase classname="testNG.TestNGAnnotations" name="m6" time="0.002"/>
  <testcase classname="testNG.TestNGAnnotations" name="m7" time="0.002"/>
  <testcase classname="testNG.Day2" name="l3" time="0.003"/>
  <testcase classname="testNG.Day2" name="m90" time="0.016">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testNG.Day2.m90(Day2.java:53)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- m90 -->
  <testcase classname="testNG.Day2" name="m90" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testNG.Day2.m90(Day2.java:53)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- m90 -->
  <testcase classname="testNG.Day2" name="m90" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testNG.Day2.m90(Day2.java:53)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- m90 -->
  <testcase classname="testNG.Day1" name="m1" time="0.001"/>
  <testcase classname="testNG.Day1" name="m2" time="0.002"/>
</testsuite> <!-- Personal Loan -->
